genapkovl-lima.sh

```diff
@@ -222,6 +235,38 @@ if [ "${LIMA_INSTALL_NERDCTL}" == "true" ]; then
     done
 fi
 
+if [ "${LIMA_INSTALL_CONTAINERD}" == "true" ] && [ "${LIMA_INSTALL_BUILDKIT}" == "true" ]; then
+    makefile root:root 0755 "$tmp/etc/init.d/buildkit" << EOF
```

Comment:
This `buildkit.openrc` file should not be inlined in the script, but be a separate file in the repo. Take a look at how `lima-init.openrc` etc are handled.